interpreter/cling/include/cling/Utils/Utils.h

```diff
@@ -0,0 +1,38 @@
+//--------------------------------------------------------------------*- C++ -*-
+// CLING - the C++ LLVM-based InterpreterG :)
+// author: Guilherme Amadio <amadio@cern.ch>
+//
+// This file is dual-licensed: you can choose to license it under the University
+// of Illinois Open Source License or the GNU Lesser General Public License. See
+// LICENSE.TXT for details.
+//------------------------------------------------------------------------------
+
+#include <cstdlib>
+#include <cstring>
+
+#ifndef CLING_UTILS_UTILS_H
+#define CLING_UTILS_UTILS_H
+
+#if defined(_MSC_VER) && !defined(strcasecmp)
+#define strcasecmp _stricmp
+#endif
+
+namespace cling {
+  namespace utils {
+    /** Convert @p value to boolean */
+    static inline bool ConvertEnvValueToBool(const char* value) {
+      const char* true_strs[] = {"1", "true", "on", "yes"};
+
+      if (!value)
+        return false;
+
+      for (auto str : true_strs)
+        if (strcasecmp(value, str) == 0)
+          return true;
+
+      return false;
```

Comment:
This seems nicer for the 👀  ;-)
```suggestion
    static inline bool ConvertEnvValueToBool(llvm::StringRef value) {
      if (value.empty())
        return false;

      for (auto str : {"1", "true", "on", "yes"})
        if (value.equals_insensitive(str))
          return true;

      return false;
```

Reply:
I think I'd combine this with `getenv` (or rather [`llvm::sys::GetEnv`](https://llvm.org/doxygen/classllvm_1_1sys_1_1Process.html#a210453e575b2fc9a4a3e10bf8a9340fd)) in one function, to be called as `getEnvBool("CLING_PROFILE")` or similar.

Reply:
Please see [this](https://github.com/root-project/root/pull/10890#discussion_r1013441471) discussion on `llvm::sys::GetEnv`.

Reply:
As a further comment, `StringRef` won't allow a `nullptr`, which I want to accept here as a false value (for when the environment variable is unset). I also thought I'd avoid the extra includes required for using a string type.